embedded-elasticsearch/src/main/java/com/playtika/test/elasticsearch/rest/WaitForGreenStatus.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Playtika
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.playtika.test.elasticsearch.rest;
+
+import lombok.RequiredArgsConstructor;
+
+import com.playtika.test.common.checks.AbstractCommandWaitStrategy;
+import com.playtika.test.elasticsearch.ElasticSearchProperties;
+
+@RequiredArgsConstructor
+public class WaitForGreenStatus extends AbstractCommandWaitStrategy {
+
+    private final ElasticSearchProperties properties;
+
+    @Override
+    public String[] getCheckCommand() {
+        return new String[]{
+            "curl", "-X", "GET",
+            "http://127.0.0.1:" + properties.getHttpPort() + "/_cluster/health?wait_for_status=yellow&timeout=10s"
```

Comment:
for what this timeout? timeout=10s

Reply:
for status
there is a param wait_for_status=yellow